app/jobs/delete_job.rb

```diff
@@ -0,0 +1,9 @@
+class DeleteWorker < ActiveJob::Base

```

Comment:
A name like `DeleteCataloguesJob` would reveal the intention of this class more clearly.